config/esp32/components/chip/Kconfig

```diff
@@ -108,6 +108,15 @@ menu "CHIP Core"
             help
                 Build CHIP test binaries.
 
+        config DISPATCH_EVENT_LONG_DISPATCH_TIME_WARNING_THRESHOLD_MS
+            int  "Set threshold in ms"
+            range 0 500
```

Comment:
IMO let's remove the range altogether, so that users can set the value as per their choice.

Also, let's set the default to around 800, as per our experiments today

Reply:
Made the suggested change.